pkg/exporters/k8sexporter/k8s_exporter.go

```diff
@@ -59,7 +59,7 @@ func NewExporterOrDie(ctx context.Context, npdo *options.NodeProblemDetectorOpti
 
 	ke := k8sExporter{
 		client:           c,
-		conditionManager: condition.NewConditionManager(c, clock.RealClock{}, npdo.K8sExporterHeartbeatPeriod),
+		conditionManager: condition.NewConditionManager(c, clock.WithTicker(clock.RealClock{}), npdo.K8sExporterHeartbeatPeriod),
```

Comment:
I don't think this change is needed.
```suggestion
		conditionManager: condition.NewConditionManager(c, clock.WithTicker(clock.RealClock{}), npdo.K8sExporterHeartbeatPeriod),
```

Reply:
yes. Thanks!